### DISCUSSION TITLE
Blank Page

### DISCUSSION THREAD

**Discussion by MidhiaQueiroz** [community/community#67139] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Why git hub home page is blank and I cant even access my profile and my repos, please, what might be the issue?

**Comment by Zerotask** [comment]
Which URL are you trying to access? Can you provide more information please and a screenshot?

I can access your [profile](https://github.com/MidhiaQueiroz)  and [repos](https://github.com/MidhiaQueiroz?tab=repositories) without any problems.

**Comment by mehulrajdev** [comment, marked as answer]
Everything on GitHub status looks fine:
https://www.githubstatus.com/

There are a few incidents in history, which I think might still be in effect even tho marked fixed.
https://www.githubstatus.com/history

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
